### ISSUE TITLE
Drifts not detected

### ISSUE THREAD

**Issue by fredochieng** [beelit94/python-terraform#120]
I have configured everything right and provisioned my AWS infrastructure with 3 instances. When I make changes to the instances in the AWS console, when I call tf.plan() method I get the output below

**No changes. Your infrastructure matches the configuration.

Terraform has compared your real infrastructure against your configuration

and found no differences, so no changes are needed.**

But when I run terraform plan in the directory where I have the terraform file (not using Python), the drifts are detected.
So in short, tf.plan() is not detecting drifts even though there are drifts. Kindly assist
Thanks
